webpack.config.js

```diff
@@ -0,0 +1,35 @@
+// Load the default @wordpress/scripts config object
+const defaultConfig = require( '@wordpress/scripts/config/webpack.config' );
+
+// Legacy jQuery UI powered admin files
+const jQueryUIAdminFileNames = [
+	'google-product-category-fields',
+	'infobanner',
+	'metabox',
+	'modal',
+	'orders',
+	'product-categories',
+	'product-sets-admin',
+	'products-admin',
+	'settings-commerce',
+	'settings-sync',
+];
+
+const jQueryUIAdminFileEntries = {};
+
+jQueryUIAdminFileNames.forEach( ( name ) => {
+	jQueryUIAdminFileEntries[ `admin/${ name }` ] = `./assets/js/admin/${ name }.js`;
+} );
+
+module.exports = {
+	...defaultConfig,
+	entry: {
+		// Use admin/index.js for any new React-powered UI
+		'admin/index.js': './assets/js/admin/index.js',
```

Comment:
Spotted an issue with this when testing #1947. The entry point config maybe should be `index` (no `.js` extension); the generated file is `index.js.js` 😆 

<img width="258" alt="Screen Shot 2021-05-13 at 3 25 18 PM" src="https://user-images.githubusercontent.com/4167300/118073172-6fe14a00-b3ff-11eb-9734-db0e328a5438.png">

Reply:
Oops, will open a PR to fix